nuxeo-services/nuxeo-platform-filemanager-core/src/main/java/org/nuxeo/ecm/platform/filemanager/service/extension/AbstractFileImporter.java

```diff
@@ -182,6 +184,8 @@ public DocumentModel create(CoreSession session, Blob content, String path, bool
             docType = getDefaultDocType();
         }
         doSecurityCheck(session, path, docType, typeService);
+        // first normalize input, as unicode can be decomposed (MacOSX behavior on WebDav)
+        fullname = Normalizer.normalize(fullname, Form.NFC);
         String filename = FileManagerUtils.fetchFileName(fullname);
```

Comment:
`FileManagerUtils.getExistingDocByPropertyName` also does normalization. So I guess the above normalization would be better put directly inside `FileManagerUtils.fetchFileName` for consistency...